examples/go-mq-basic/pullrouter.go

```diff
@@ -64,26 +46,27 @@ func (mq *pullMQRouter) run(ctx context.Context) {
 }
 
 func (mq *pullMQRouter) reload(ctx context.Context) {
-	mq.Log.Debug("reloading routes", "baseline", mq.baseline)
+	mq.Log.Debug("reloading routes", "baseline", mq.Baseline)
+
 	// load routes from route server
-	resp, err := mq.grpcClient.GetWorkloadRoutes(ctx, &routesapi.WorkloadRoutesRequest{
-		BaselineWorkload: mq.baseline,
-	})
+	resp, err := mq.getRoutes()
 	if err != nil {
 		mq.Log.Error("couldn't get workload routes", "error", err)
 		return
 	}
 
-	// recompute the routing map
-	routingMap := make(map[string]string, len(resp.Routes))
+	// collect received routing keys
+	rkSet := set.New()
+	rkList := make([]string, 0, len(resp.Routes))
```

Comment:
This list is unneeded?

Reply:
I defined a helper function to print sets (`set2String()`), and got rid of that slice.